addons/html_editor/static/src/components/history_dialog/history_dialog.scss

```diff
@@ -1,168 +1,276 @@
-                line-height: 24px;
-                background-color: $secondary;
-                z-index: 10;
+                font-size: .5rem;
+                padding-left: 2px;
+                color: white;
+                background-color: $o-action;
             }
-            &.targeted {
-                color: lighten($primary, 20%);
-                &:before {
-                    color: white;
-                    content: '\f00c';
-                    background-color: lighten($primary, 20%);
-                }
-                &:after {
-                    border-color: lighten($primary, 20%);
-                }
+
+            ~ .list-group-item:before {
+                background-color: $o-action;
             }
-            &.selected {
-                color: $primary;
-                &:before {
-                    color: white;
-                    content: '\f0da';
-                    background-color: $primary;
-                }
-                &:after {
-                    border-color: $primary;
-                }
-                .o_avatar {
-                    opacity: 1;
-                }
+        }
+
+        &:first-child:after {
+            content: none;
+        }
+    }
+}
+
+@include media-breakpoint-down(md) {
+    .html-history-dialog {
+        flex-direction: column;
+
+        .revision-list {
+            min-width: 100%;
+            width: 100%;
+            flex: 1;
+            border-right: none;
+            overflow-y: auto;
+        }
+
+        .history-container {
+            flex: 1;
+            min-height: 0;
+        }
+
+        &.mobile-show-revisions {
+            .revision-list {
+                display: block;
             }
-            &:first-child:after {
-                content: none !important;
+            .history-container {
+                display: none;
             }
+        }
+
+        &.mobile-show-content {
+            .revision-list {
+                display: none;
+            }
+            .history-container {
+                display: flex;
+                flex-direction: column;
+            }
+        }
+    }
 
+    .nav-tabs {
+        background-color: $o-view-background-color;
+
+        .nav-link {
+            background-color: $o-gray-100;
+            border-bottom-color: $border-color;
+
+            &.active {
+                --#{$prefix}nav-tabs-link-active-color: var(--historyDialog__link-color--active, #{$o-brand-odoo});
+                border-bottom-color: $o-view-background-color;
+
+                &::after {
+                    position: absolute;
+                    top: -$border-width;
+                    left: -$border-width;
+                    right: -$border-width;
+                    height: $border-width * 3;
+                    background-color: var(--historyDialog__link-border-top-color--active, #{$o-brand-odoo});
+                    content: '';
+                }
+            }
```

Comment:
These are not scoped under the `.html-history-dialog` the style applies to all the `.nav-tabs` < md